src/request_handler.rb

```diff
@@ -0,0 +1,66 @@
+require 'yaml'
+
+class RequestHandler
+  attr_reader :message, :bot, :receiver
+
+  def initialize(receiver)
+    @receiver = receiver
+    @message = receiver.message
+    @bot = receiver.bot
+
+    @answers = YAML.load_file(File.dirname(__FILE__) + '/data/messages.yml')
+
+    handle_message
+  end
+
+  def handle_message
+    case message.text
+      when 'What time is it?'
```

Comment:
[Usually](https://github.com/bbatsov/ruby-style-guide#indent-when-to-case) we aligning `when` as deep as `case`.